### PR TITLE
Feature email confirmation #27

### PR DESCRIPTION
**Pull request by LKhyeon**
### What this PR does (required):
- When the user creates an account, creates an appointment, or deletes an appointment, send email to the user, and the appointee (if appointment creation/deletion).

### Screenshots / Videos (required):
- Email sent upon appointment creation.
image

Clicking on the link:
image

- Email sent upon appointment deletion
image

- Email sent upon account creation
image 

### Any information needed to test this feature (required):
- Add SENDGRID_API_KEY, SENDGRID_FROM_EMAIL, DEV_URL, PROD_URL from slack group chat to env file.
- cd `/client` and run `npm install` (also, might need to run `npm update --force` if running into chokidar version issue). Then run `npm start`.
- cd `/server` and run `npm install`. Then run `npm run dev`.
- To test out account creation, email try creating a new account using a google account that hasn't been used yet.
- To test out appointment creation and appointment deletion email, go to `localhost:3000/shared/test/60f323381a376a60c971ca4a` and use the account pinned on the slack group chat to check the email sent to the host.

### Any issues with the current functionality (optional):
- All of the functionalities are there, but as of right now, there are no styles in the email themselves. I plan to make a change in the future to style them given enough time after finishing other necessary tasks.

### CODE REVIEW

File: server/controllers/appointment.js [hunk 1]
```diff
@@ -41,6 +42,9 @@ exports.createAppointment = asyncHandler(async (req, res, next) => {
     );
   }
 
+  // Send Emails
+  appointCreateDeleteEmail(true, true, appointment);
+  appointCreateDeleteEmail(true, false, appointment);
```
Comment (ajaytomgeorge) [inline server/controllers/appointment.js]: This seems a little unnecessary, appointCreateDeleteEmail has been called twice with toHost set to true and false, which effectively makes that variable redundant.  And using too much Boolean flags can be confusing,  You can simplify it into two functions createMail and DeleteMail or any other simplified version u prefer.

File: server/controllers/appointment.js [hunk 2]
```diff
@@ -71,6 +75,9 @@ exports.deleteAppointmentById = asyncHandler(async (req, res, next) => {
     throw new Error("No appointment found with given id");
   }
 
+  // Send Emails
+  appointCreateDeleteEmail(false, true, deleted);
+  appointCreateDeleteEmail(false, false, deleted);
```
Comment (ajaytomgeorge) [inline server/controllers/appointment.js]: Same as above

**Comment by ajaytomgeorge** [comment]
Good work

**Review by moffatethan — APPROVED** [review]
Looks good Luke, thank you!

**Pull request merged**